BastionUserOnboarding-SSH-over-SSM.md

```diff
@@ -144,9 +150,14 @@ Host *.probation.hmpps.dsd.io !*.stage.delius.probation.hmpps.dsd.io !*.pre-prod
 
 * Mac/Linux
 
-Replace `YOUR_USER_NAME_HERE` with your ssh username. This is usually `<first initial><surname>`, e.g. jbloggs
+Replace `YOUR_USER_NAME_HERE` with your ssh username. This is usually `<first initial><surname>`, e.g. jbloggs<br />
+Replace `PROD_BASTION_INSTANCE_ID` with the instance id. You can retrieve the dev bastion instance id by running
```

Comment:
prod used in token, but then eng dev is mentioned

Reply:
thanks Pete, resolved it